### PR TITLE
Add PR Template to Repository

### PR DESCRIPTION
**Pull request by laurapacilio**
Adding the PR template to help guide contributors.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+### Merge Checklist
+_If items do not apply to your changes, add (N/A) and mark them as complete._
+
+#### Pull Request
+- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
+- [ ] Description links to related pull requests or issues, if any.
+
+#### Content
+- [ ] Redirects have been added for moved, renamed, or deleted pages.
```
Comment (thiskevinwang) [inline .github/pull_request_template.md:20]: Just a callout:

Sub project redirects are not yet supported. They need be handled in https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js

We do have an Asana task for this though!

**Review by thiskevinwang — APPROVED** [review]
1 callout, otherwise looks good!

File: .github/pull_request_template.md (hunk 1 above)
Comment (laurapacilio) [reply .github/pull_request_template.md:20]: OHHH okay thank you!! let me add some further explanation :)

Comment (laurapacilio) [inline .github/pull_request_template.md:20]:
```suggestion
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
```

**Pull request merged**